### PR TITLE
chore: add Dependabot for version updates

### PR DESCRIPTION
**Pull request by nschonni**
Now the CI is testing across platforms, Dependabot can be used to start testing dependdency updates

### CODE REVIEW

**Comment by aearly** [comment]
Personally, I find dependabot annoying.  I'd prefer to do this kind of thing manually a few times a year at most, especially since we only have dev dependencies.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: '/'
+    schedule:
+      interval: weekly
```
Comment (nschonni) [inline .github/dependabot.yml:6]: Could move this to monthly to reduce the noise

Comment (aearly) [reply .github/dependabot.yml:6]: Maybe bi-annually?  All deps are devDeps, so it's not critical to keep everything up to date.

**Review by aearly — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (nschonni) [reply .github/dependabot.yml:6]:
```suggestion
      interval: monthly
```

**Comment by nschonni** [comment]
I bumped it up to Monthly, but that's the make interval right now

**Pull request merged**